track/templates/includes/fr/header.html

```diff
@@ -1,7 +1,7 @@
 <div class="container mx-auto items-center sm:w-4/5 xl:w-3/5 mt-6">
 	<h1 class="text-4xl sm:text-5xl">Vérifier la conformité</h1>
 	<h2 class="mb-4 sm:mb-6 text-xl sm:text-2xl">Dernière mise à jour {{ scan_date | display_date("fr") }}</h2>
-	<p class="text-lg mb-6">Ce tableau indique dans quelles proportions les sites et services en ligne gouvernementaux respectent les bonnes pratiques en matière de sécurité Web décrites dans <a class="text-https-blue hover:text-black font-bold" href="https://www.canada.ca/fr/secretariat-conseil-tresor/services/technologie-information/avis-mise-oeuvre-politique/mise-oeuvre-https-connexions-web-securisees-ampti.html">l’Avis de mise en œuvre de la Politique sur la technologie de l’information (AMPTI)&nbsp;: Mise en œuvre de HTTPS pour les connexions Web sécurisées</a>.</p>
-	<p class="text-lg mb-8"><a id="modal-btn" class="font-bold text-https-blue hover:text-black" href="#">Comment interpréter ce tableau?</a></p>
+	<p class="text-lg mb-6">Ce tableau indique dans quelles proportions les sites et services en ligne gouvernementaux respectent les bonnes pratiques en matière de sécurité Web décrites dans <a class="text-https-blue hover:text-black " href="https://www.canada.ca/fr/secretariat-conseil-tresor/services/technologie-information/avis-mise-oeuvre-politique/mise-oeuvre-https-connexions-web-securisees-ampti.html">l’Avis de mise en œuvre de la Politique sur la technologie de l’information (AMPTI)&nbsp;: Mise en œuvre de HTTPS pour les connexions Web sécurisées</a>.</p>
+	<p class="text-lg mb-8"><a id="modal-btn" class=" text-https-blue hover:text-black" href="#">Comment interpréter ce tableau?</a></p>
```

Comment:
For readability of code, I'd remove the extra white space where font-bold was removed. Otherwise it looks good to me!

Reply:
aaah we do need a second set of eyes :)    

Unwanted spaces are now removed.